R/reportSE.R

```diff
@@ -60,14 +60,17 @@ reportSE <- function(gdx, regionSubsetList = NULL, t = c(seq(2005, 2060, 5), seq
   input_liquids <- c(entyPe, "seh2")
 
   ## parameter
-  pm_prodCouple    <- readGDX(gdx, c("pm_prodCouple", "p_prodCouple", "p_dataoc"), restore_zeros = FALSE, format = "first_found")
+  pm_prodCouple <- readGDX(gdx, c("pm_prodCouple", "p_prodCouple", "p_dataoc"), restore_zeros = FALSE, format = "first_found")
+  pm_prodCoupleEmi <- readGDX(gdx, "pm_prodCoupleEmi", restore_zeros = FALSE, react = "silent")
```

Comment:
does pm_prodCoupleEmi also exist in the old version?

Reply:
no. `readGDX` will return `NULL` (without warning because of `react = "silent"`)